README.md

```diff
@@ -374,12 +374,11 @@ if let foo = foo {
 Or when unwrapping multiple optionals, prefer this:
 
 ```swift
-if 
-    let foo = foo,
-    let bar = bar {
-    	// Indent once more to use `foo` and `bar` to show a new scope
-        foo.bizz()
-        bar.bazz()
+if let foo = foo.optionalProperty as? SomeType,
+    let bar = bars.filter({ $0.isMyBar }).first
+    { // Put the open bracket on a new line
```

Comment:
This fights the default Xcode indentation, and so does the previous style.

How about instead do this but indent it to what Xcode prefers which is
```
if let foo ...,
    let bar = ...
{
    foo.bizz()
    bar.bazz()
}
```